### PR TITLE
Update processor for doctrine/persistence:^2.0

### PR DESCRIPTION
**Pull request by lepiaf**
Hello,

I update processor according change from swarrot bundle https://github.com/swarrot/SwarrotBundle/pull/202

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
     "require-dev": {
         "phpunit/phpunit": "^8.4",
         "php-amqplib/php-amqplib": "^2.1",
-        "doctrine/common": "^2.3",
-        "doctrine/dbal": "^2.4",
+        "doctrine/common": "^3.0",
```
Comment (stof) [inline composer.json:21]: this should not force the 3.0 version IMO, as the latest 2.x versions of common are still compatible. This should be using a `||` in the constraint.

**Review by stof — CHANGES_REQUESTED** [review]

**Comment by mm-pvgnd** [comment]
@odolbeau I though this MR has been already merged as the one in SwarrotBundle was. Could you merge & release please? Because since we released SwarrotBundle 2.1.0, this change in swarrot is missing.

File: composer.json (hunk 1 above)
Comment (mm-pvgnd) [reply composer.json]: @lepiaf has changed this to 2.9, maybe we can allow 3.0 in Swarrot & SwarrotBundle in other MRs

Comment (odolbeau) [reply composer.json]: @stof is it OK for you?

**Pull request merged**